ChangeLog

```diff
@@ -1,3 +1,7 @@
+2020-10-24  Condrad Sanderson  <conradsand@gmail.com>
```

Comment:
Please remove this email address.  If an email address is really necessary, I'm okay for the associated minor change in code to be attributed to to you, or perhaps "RcppArmadillo community" in general.

Reply:
No that's what we do. Changes get a ChangeLog entry.